src/taipy/gui/gui_actions.py

```diff
@@ -220,16 +220,18 @@ def invoke_long_callback(
         user_status_function (Optional(Callable[[State^, bool, ...], None])): The optional user-defined status
             function that is invoked at the end of and possibly during the runtime of *user_function*:
 
-            - The first parameter of this function **must** be a `State^`.
-            - The second parameter of this function **must** be a bool or an int.<br/>
+            - The first parameter of this function will be set to a `State^` instance.
+            - The second parameter of this function will be set to a bool or an int, depending on the
+              conditions under which it is called:
 
-               - If the status function receives a Boolean value, then:
+               - If the status function receives a bool value in its second parameter, then:
```

Comment:
We're already in the second parameter context here ;-)

Reply:
replace "If the status function receives ..."
by
"If this parameter is set to ..." ?